GameDefinition/config.ui.default

```diff
@@ -15,6 +15,25 @@
 [additional_commands]
 ; Angband compatibility (accept target)
 Cmd_2 = ("KP_Insert", ([CmdAim], "", ByAimMode AimModeCmd {exploration = ExecuteIfClear Dashboard, aiming = Accept}))
+; Custom key rebinding example:
+;
+;   CmdItem = ("a", ([], "My Happy Cmd", Macro["t"]))
```

Comment:
This is fine, but actually I meant `CmdItem` instead of `CmdAim` below.

Reply:
I mean, below, all occurrences. 'CmdItem probably matches it better, since that's the category used for "t"'